templates/common.jade

```diff
@@ -0,0 +1,19 @@
+extends /templates/layout
+
+block body
+    if task == "edit"
+        .container#content
+            #relationshipForm
+    else if task == "delete"
+        h1 Delete Entity
```

Comment:
Missing the container here:

    .row.margin-top-2
        #deletion-form!= markup